org.eclipse.jdt.ls.tests/pom.xml

```diff
@@ -85,6 +85,11 @@
 										<version>0.0.0</version>
 										<type>eclipse-feature</type>
 									</dependency>
+									<dependency>
+										<artifactId>org.eclipse.jdt.java17patch</artifactId>
+										<version>0.0.0</version>
+										<type>eclipse-feature</type>
+									</dependency>
```

Comment:
I don't think this is necessary now that we consume from the upstream main branch

Reply:
Fixed.